start_qemu_mixed-firmware.sh

```diff
@@ -15,4 +15,10 @@ root="$dir"
 
 image="${root}/deploy/mixed-firmware/Syslinux_Linuxboot.img"
 
-qemu-system-x86_64 -drive if=virtio,file=${image},format=raw -net user -net nic -device virtio-rng-pci -rtc base=localtime -m 8192 -nographic 
+qemu-system-x86_64 \
+  -drive if=virtio,file=${image},format=raw \
+  -nographic \
+  -net user,hostfwd=tcp::2222-:2222 \
+  -net nic \
+  -object rng-random,filename=/dev/urandom,id=rng0 \
```

Comment:
what is the difference to `-device virtio-rng-pci`?

Reply:
I don't actually know. some of the options were copied over from the cpu repo.

Reply:
I believe that this is the provider for the device since they get the same ID (rng0). Please take a look at the line below